include/database.py

```diff
@@ -32,3 +33,32 @@ def select(connectionString, sql, parameters):
     except psycopg2.DatabaseError as err:
         sys.stderr.write(f'A general database error occurred: {err}')
         raise DatabaseError
+
+
+# db connection to get studyId
+def insert_returning(connectionString, sql, parameters):
+
+    try:
+        # Open the connection, override any timeout provided with something shorter
+        # since we expect to be running interactively
+        connection = psycopg2.connect(connectionString, connect_timeout=1)
+        cursor = connection.cursor()
+
+        # Execute the query, note the rows
+        cursor.execute(sql, (parameters['Name'],))
+        study_id = cursor.fetchone()[0]
```

Comment:
Suggest `returnValue` instead of `study_id`.

Side note, but as a matter of convention usually I've been writing function names with underscores and variable names as camel case.